compiler/rustc_target/src/spec/targets/aarch64_pc_windows_msvc.rs

```diff
@@ -7,7 +8,7 @@ pub fn target() -> Target {
 
     Target {
         llvm_target: "aarch64-pc-windows-msvc".into(),
-        description: None,
+        description: Some(Cow::Borrowed("ARM64 Windows MSVC")),
```

Comment:
Can you use into instead like the other values?

Reply:
Done